docker-compose.yml

```diff
@@ -89,6 +89,17 @@ services:
     - "127.0.0.1:6391:6379"
 
 # Dependencies
+  cosmosdb-emulator:
+    image: mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator:latest
+    container_name: cosmosdb-emulator
+    environment:
+      - AZURE_COSMOS_EMULATOR_PARTITION_COUNT=2
```

Comment:
I think the default is 10

But we only have `LastName` as a partition key, so decreasing seems fine IMO